### ISSUE TITLE
bug: Dependency Conflict: nanopb Version Mismatch

### ISSUE THREAD

**Issue by jonasbehlic** [capawesome-team/capacitor-plugins#332]
### Plugin(s)

- [ ] Android Foreground Service
- [ ] Android Battery Optimization
- [ ] App Review
- [ ] App Update
- [ ] Background Task
- [ ] Badge
- [ ] Bluetooth Low Energy
- [ ] Cloudinary
- [ ] Datetime Picker
- [ ] File Compressor
- [ ] File Opener
- [ ] File Picker
- [X] Live Update
- [ ] Managed Configurations
- [ ] NFC
- [ ] Photo Editor
- [ ] Printer
- [ ] Screen Orientation
- [ ] Speech Recognition
- [ ] Speech Synthesis
- [ ] Torch
- [ ] Zip

### Version

6.2.0

### Platform(s)

- [X] Android
- [X] iOS
- [ ] Web

### Current behavior

Here's a well-structured GitHub issue description for the dependency conflict you're encountering:

Dependency Conflict: nanopb Version Mismatch
Description
When installing the dependencies for my Capacitor project, I encountered a version conflict involving the nanopb dependency. The issue arises from incompatible version constraints required by different Capacitor plugins.

Conflict Details
Current Podfile.lock:
nanopb (< 2.30911.0, = 2.30910.0, >= 2.30908.0)

Dependencies:

CapacitorFirebaseMessaging (@capacitor-firebase/messaging, version 6.2.0)

Depends on: FirebaseMessaging (~> 11.0)
Resolved to: FirebaseMessaging 11.5.0
Which depends on: nanopb (~> 3.30910.0)
CapacitorMlkitBarcodeScanning (@capacitor-mlkit/barcode-scanning, version 6.2.0)

Depends on: GoogleMLKit/BarcodeScanning (= 5.0.0)
Which depends on: GoogleMLKit/MLKitCore (= 5.0.0)
Which depends on: MLKitCommon (~> 10.0.0)
Which depends on: GoogleDataTransport (~> 9.0)
Resolved to: GoogleDataTransport 9.4.1
Which depends on: nanopb (< 2.30911.0, >= 2.30908.0)

### Expected behavior

No mismatch

### Reproduction

https://github.com/capawesome-team/capacitor-plugins/

### Steps to reproduce

Add the following plugins to a Capacitor project:
@capacitor-firebase/messaging version 6.2.0
@capacitor-mlkit/barcode-scanning version 6.2.0
Run pod install in the iOS project directory.
Observe the CocoaPods could not find compatible versions for pod "nanopb" error.

### Capacitor doctor

Latest Dependencies:

  @capacitor/cli: 6.2.0
  @capacitor/core: 6.2.0
  @capacitor/android: 6.2.0
  @capacitor/ios: 6.2.0

Installed Dependencies:

  @capacitor/cli: 6.2.0
  @capacitor/core: 6.2.0
  @capacitor/android: 6.2.0
  @capacitor/ios: 6.2.0

### Before submitting

- [X] I have read and followed the [bug report guidelines](https://capawesome.io/contributing/bug-reports/).
- [X] I have attached links to possibly related issues and discussions.
- [X] I understand that incomplete issues (e.g. without reproduction) are closed.

**Comment by robingenz** [comment]
I close this as duplicate of https://github.com/capawesome-team/capacitor-firebase/issues/750.
